src/test/scala/org/camunda/feel/impl/interpreter/InterpreterFunctionTest.scala

```diff
@@ -179,4 +179,8 @@ class InterpreterFunctionTest
     eval("round(3.2)", functions = functions) should be(ValNumber(3))
   }
 
+  it should "be properly evaluated when parameters contain whitespaces" in {
```

Comment:
:x: The test case should be grouped under `"A function definition"`.

Currently, it is grouped under `"An external java function definition"` that is the last test group in this class. We should move the test case up, for example, after `it should "be invoked and followed by a filter"`. It is not related to external functions.

![image](https://user-images.githubusercontent.com/4305769/147914741-ebc283bc-7e97-4354-b45a-e4127cd49c8a.png)

Reply:
my bad, gonna fix it!